src/main.test.js

```diff
@@ -36,15 +36,15 @@ test("Valid input with default value for type", () => {
 
 test("Valid input with default value for restricted choices", () => {
   const testEnv = "DATABASE_URL: email = test@mail.com";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
     `<label for="env_spec_database_url">DATABASE_URL</label>\n` +
       `<input id="env_spec_database_url" name="database_url" type="email" value="test@mail.com" />\n`
   );
 });
 
 test("Valid input with comment,middle of line", () => {
   const testEnv = "DATABASE_URL: email = test@mail.com #comment";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#resolves)).